### PR TITLE
feat: support check dependency service status for init container

### PR DESCRIPTION
**Pull request by schneesu**

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-bcs?pullRequest=1705) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-bcs?pullRequest=1705) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-bcs?pullRequest=1705) it.</sub>

File: bcs-services/cluster-resources/cmd/check.go [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+	"github.com/Tencent/bk-bcs/bcs-services/cluster-resources/pkg/cache/redis"
+	"github.com/Tencent/bk-bcs/bcs-services/cluster-resources/pkg/cluster"
+	"github.com/Tencent/bk-bcs/bcs-services/cluster-resources/pkg/config"
+	"github.com/Tencent/bk-bcs/bcs-services/cluster-resources/pkg/project"
+	"github.com/Tencent/bk-bcs/bcs-services/cluster-resources/pkg/util/stringx"
+)
+
+const (
+	// 重试次数
+	RetryTimes = 10
```
Comment (dellkeji) [inline bcs-services/cluster-resources/cmd/check.go:37]: 这几个需要放到配置吗？

File: bcs-services/cluster-resources/cmd/cr.go [hunk 2]
```diff
@@ -59,6 +60,11 @@ func Start() {
 	logger.Info(fmt.Sprintf("ConfigFilePath: %s", *confFilePath))
 	logger.Info(fmt.Sprintf("VersionBuildInfo: {%s}", version.GetVersion()))
 
+	// 若指定了只检查依赖服务，则检查通过后以零值退出，否则以非零值退出
+	if *checkService {
+		NewDependencyServiceChecker(crConf).DoAndExit()
```
Comment (dellkeji) [inline bcs-services/cluster-resources/cmd/cr.go:65]: 是否默认检测一次，但是不退出？

File: bcs-services/cluster-resources/cmd/check.go [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+	)
+	if err = microRtr.Init(); err != nil {
+		panic(fmt.Sprintf("init microRtr failed: %v", err))
+	}
+	return microRtr
+}
+
+func initCliTLSConf(conf config.ClientConf) *tls.Config {
+	if conf.Cert != "" && conf.Key != "" && conf.Ca != "" {
+		tlsConf, err := ssl.ClientTslConfVerity(conf.Ca, conf.Cert, conf.Key, conf.CertPwd)
```
Comment (dellkeji) [inline bcs-services/cluster-resources/cmd/check.go:91]: Nit: 这里后续可以添加支持加密密码

**Pull request merged**